src/CSSTransitionGroup.js

```diff
@@ -53,7 +37,21 @@ class CSSTransitionGroup extends React.Component {
   }
 }
 
-CSSTransitionGroup.propTypes = propTypes;
-CSSTransitionGroup.defaultProps = defaultProps;
+CSSTransitionGroup.propTypes = {
```

Comment:
The plugin traverses the AST using the default parser... I think it's Babylon under the hood. It wraps the direct node and can't reverse engineer because it would require to find all references I believe which is very complicated.